index.html

```diff
@@ -188,29 +188,20 @@ <h3>Skills</h3>
 </ul>
 </section>
 <footer class="footer-cont">
-    <h2 class="contact">Contact Me</h2>
+    <h2 class="contact1">Contact Me</h2>
     <p class="footerpara">
         If you have an application you are interested in developing,
          a feature that you need built or a project that needs coding. 
          I’d love to help with it 
     </p>
-    <form  action="actionpage.php" class='contactform' name="contactfrm" method="POST">
-       <div class="form-continer">
-            <input type="text" placeholder="Yeremias" required="required" maxlength="40" class="input"
-                name="name" id="name">
-        <br><br>
-            <input type="email" placeholder="natajayanj14@gmail.com" required="required" class="input"
-                name="email" id="email">
-            <br /><br>
-            <textarea placeholder="Write your Message here" required="required" class="input" maxlength="500"
-                name="message" id="description" cols="30" rows="10"></textarea>
-        <br><br>
-        <div class="submit-btn">
-            <button type="submit" class="sub-btn">  Get in Touch
-            </button>
-        </div>
-    </div>
-    </form>
+    <!-- form-->
+    <form action="https://formspree.io/f/xwkyolko" method="POST" class="form-cont">
+          <input type="text" placeholder="Yeremias" required="required" maxlength="40" class="input" name="name" id="name">
+          <input type="email" placeholder="natajayanj14@gmail.com" required="required" class="input" name="email" id="email">
+          <textarea placeholder="Write your Message here" required="required"
+           class="input" maxlength="500" name="message" id="description"></textarea>
+          <button class="form-button" type="submit" id="submit">Get in touch</button>
+      </form>
 </footer>
```

Comment:
- [ ] You have built the contact section in a good way 👍. Let's improve it by removing the margins from the left, right, and the bottom of your contact section to improve the UI design and also to match the Figma design.
- Your design:
<img width="212" alt="image" src="https://user-images.githubusercontent.com/77190203/154130591-b1b396c3-36f7-4316-9d09-4db5c95820b1.png">

Reply:
@medaminedev66 
Thank you for the great review i have fixed the problem.